### PR TITLE
Update instructions on how to add widget to page

### PR DESCRIPTION
**Pull request by martypowell**
readme update

### CODE REVIEW

File: js/featured-events-widget.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # Featured Events Widget
 
-## Markup
-
+## Getting Started
+
+1. Go to the site executive admin console
```
Comment (mxsnyder) [inline js/featured-events-widget.md]: SiteExecutive is one word

File: js/featured-events-widget.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+1. Go to inclusions < shared
```
Comment (mxsnyder) [inline js/featured-events-widget.md]: Use a greater-than

File: js/featured-events-widget.md [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
+1. Download a copy of `featured-events-widget.html`
+1. Modify the featured-events element to your needs
+1. Save and upload the file to the the inclusions < shared folder with a **meaningful name** (this could be re-used!)
```
Comment (mxsnyder) [inline js/featured-events-widget.md]: Greater-than

**Comment by martypowell** [comment]
@mxsnyder one more look please

**Pull request merged**